suncal/models/astro.py

```diff
@@ -87,6 +109,19 @@ def events(self):
         sunrise = sun_events['rise']
         sunset = sun_events['set']
```

Comment:
That would raise if the key is missing. Consider using `sun_events.get(..., None)` instead.

Reply:
The keys always exist, because the function that returns `sun_events` is implemented that way.